cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -1196,6 +1239,14 @@ pub enum CoreEnvironmentError {
     OpenManifest(#[source] std::io::Error),
     #[error("could not write manifest")]
     UpdateManifest(#[source] std::io::Error),
+    /// Tried to uninstall a package that wasn't installed
+    #[error("couldn't uninstall '{0}', wasn't previously installed")]
+    PackageNotFound(String),
+    // Multiple packages match user input, must specify install_id
+    #[error(
+        "multiple packages match '{0}', please specify an install id from possible matches: {1:?}"
```

Comment:
Note to self: May want to tweak how the matches are displayed by doing a `.join(", ")` into a `String` instead